apps/src/templates/teacherDashboard/AssignmentVersionMenuItem.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component, PropTypes } from 'react';
+import PopUpMenu from "../../lib/ui/PopUpMenu";
+import {assignmentVersionShape} from './shapes';
+
+export default class AssignmentSelector extends Component {
+  static propTypes = {
+    version: assignmentVersionShape,
+    onClick: PropTypes.func.isRequired,
+  };
+
+  handleClick = () => {
+    this.props.onClick();
+  };
```

Comment:
Do we particularly care about _not_ passing the event argument to this callback?  If not, maybe simpler to just pass `this.props.onClick` to the `PopUpMenu.Item` below.